docs/create/vercel/index-vercel.mdx

```diff
@@ -0,0 +1,216 @@
+* Vercel provides custom domains to deploy your code on the live server (vercel.app as the suffix in the domain). 
+* Vercel provides you with an option to choose any framework of the repository you wish to deploy either Node.js, React, Gatsby, or [Next.js](https://nextjs.org/) (a full-stack React serverless framework that integrates seamlessly with Vercel).
+* Vercel integrates with a GitHub repository for automatic deployments upon commits. 
+
+In this tutorial, you will learn how to deploy a Node.js based Redis chat application using Vercel in just 5 minutes.
+
+
+### Table of Contents
+
+
+
+* Step 1. Set up a free Redis Enterprise Cloud account
+* Step 2. Install Vercel CLI
+* Step 3. Log in to your Vercel Account
+* Step 4. Clone your GitHub repository
+* Step 5. Create a vercel.json file
+* Step 6. Set up environment variables
+* Step 7. Deploy the Node.js app
+* Step 8. Access your app
+
+### Step 1. Set up a free Redis Enterprise Cloud account
+
+Visit [developer.redis.com/create/rediscloud/](https://developer.redis.com/create/rediscloud/) and create [a free Redis Enterprise Cloud account](https://redis.com/try-free/). Once you complete this tutorial, you will be provided with the database endpoint URL and password. Save it for future reference.
+
+
+
+
+
+![alt_text](images/vercel2.png "image_tooltip")
+
+
+### Step 2. Install Vercel CLI 
+
+
+```
+npm i -g vercel
+
+vercel -v     
+Vercel CLI 23.1.2
+23.1.2
+
+```
+
+### Step 3. Log in to your Vercel account
+
+The ```vercel login``` command allows you to log in to your Vercel account through Vercel CLI.
+
+
+```
+vercel login
+Vercel CLI 23.1.2
+> Log in to Vercel github
+> Success! GitHub authentication complete for xx@xx.com
+Congratulations! You are now logged in. In order to deploy something, run `vercel`.
+💡  Connect your Git Repositories to deploy every branch push automatically (https://vercel.link/git).
+```
+
+
+Once Vercel gets connected to your GitHub account, it displays your public repository.. Let us clone [https://github.com/redis-developer/basic-redis-chat-app-demo-nodejs](https://github.com/redis-developer/basic-redis-chat-app-demo-nodejs) to the local repository.
+
+### Step 4. Clone the GitHub repository
+
+The complete source code of Redis Chat application is hosted [here](https://github.com/redis-developer/basic-redis-chat-app-demo-nodejs). React and Socket.IO is used for building the frontend while Node.js and Redis powers the backend. Redis is used mainly as a database to keep the user/messages data and for sending messages between connected servers.
+
+```
+git clone https://github.com/redis-developer/basic-redis-chat-app-demo-nodejs
+```
+
+
+### Step 5. Create a vercel.json for Node.js app
+
+If you run ```vercel init``` command, it will list various framework but you won’t be able to find Node.js, hence you will need to create a ```vercel.json``` file as shown below:
```

Comment:
```suggestion
If you run the ```vercel init``` command, it will list various frameworks but you won’t be able to find Node.js, hence you will need to create a ```vercel.json``` file as shown below:
```